BUILD.gn

```diff
@@ -314,6 +314,7 @@ source_set("libdawn_native_sources") {
   deps = [
     ":dawn_common",
     ":libdawn_native_utils_gen",
+    "${dawn_spirv_tools_dir}:spvtools_val",
```

Comment:
Probably obvious, but I suppose this means that all Dawn clients (e.g., skia-dawn) now must have spirv-tools as a dependency?

Reply:
Yes, but it would have been the case before too if you use our BUILD.gn files.